src/site/teasers/bio.drupal.liquid

```diff
@@ -42,11 +42,12 @@ Example data:
 {% endif %}
 <div class="vads-u-display--flex vads-u-margin-bottom--4">
     {% if node.fieldMedia != empty %}
-        <div class="vads-u-margin-right--2 vads-u-flex--auto">
-            {% assign image = node.fieldMedia.entity.image %}
-            <img class="bio-image" src="{{ image.derivative.url }}" alt="{{ image.alt }}"
-                title="{{ image.title }}">
-        </div>
+    <div class="vads-u-margin-right--2 vads-u-flex--auto">
+        {% assign image = node.fieldMedia.entity.image %}
+        <img data-object-fit="cover" class="bio-image"
+            src="{{ image.derivative.url }}" alt="{{ image.alt }}"
+            title="{{ image.title }}">
+    </div>
```

Comment:
CS: Lost its indentation to the 'if' logic.

Reply:
Yeah, I'm not really sure what our standard for liquid is, @swirtSJW - we have a mixture of styles in different liquid files. My vscode liquid formatter wants to bring code blocks in conditionals flush with the conditional. Do you have a vscode tool in place that is alerting on this / formatting differently?

Reply:
I don't, my tired eyes just like the readability of nested code.

I could not find official Code style definitions but I see the example provided here 
https://shopify.github.io/liquid/basics/operators/  use what I would considered  standard.